### PR TITLE
Allow access codes delete interaction

### LINKED ISSUE #152: Cannot delete an access code on a device
Trying to reach `access-codes/delete` with an `access_code_id` returns the following error `CLI Error: TypeError: Cannot read properties of undefined (reading 'name')`

Screenshot 2024-05-07 at 15 11 37

### PR DESCRIPTION
**Pull request by kainpets**
Closes #152 
Screenshot 2024-05-15 at 11 56 04

### CODE REVIEW

**Review by andrii-balitskyi — APPROVED** [review]

File: lib/interact-for-custom-metadata.ts [hunk 1]
```diff
@@ -5,7 +5,9 @@ export const interactForCustomMetadata = async (
   custom_metadata: CustomMetadata
 ) => {
   const seam = await getSeam()
-  const updated_custom_metadata = { ...custom_metadata }
+  const updated_custom_metadata: { [key: string]: string | boolean | null } = {
```
[the remark this answers is not in the document]
Comment (kainpets) [reply lib/interact-for-custom-metadata.ts:8]: CustomMetadata doesn't allow for null which results in a type error at line 58. Guess I'll expand the type

[the remark this answers is not in the document]
Comment (andrii-balitskyi) [reply lib/interact-for-custom-metadata.ts:8]: I checked out your PR and it seems that it does allow for null
image

[the remark this answers is not in the document]
Comment (kainpets) [reply lib/interact-for-custom-metadata.ts:8]: Am I somehow in another dimension 😅 
Screenshot 2024-05-15 at 13 03 28

[the remark this answers is not in the document]
Comment (andrii-balitskyi) [reply lib/interact-for-custom-metadata.ts]: I think I had outdated @seamapi/types

**Pull request merged**